src/main/java/it/pagopa/selfcare/pagopa/model/stationmaintenance/MaintenanceHoursSummaryResource.java

```diff
@@ -0,0 +1,43 @@
+package it.pagopa.selfcare.pagopa.model.stationmaintenance;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+
+/**
+ * Model class that hold the hours' summary of stations' maintenance for a specific broker
+ */
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class MaintenanceHoursSummaryResource {
```

Comment:
specify the format for the fields of this class

Reply:
Hours are not in a specific date format, are composed of hours on base 10 and minutes on base 60. Example: 30 and half hours is "30:30", 30 and a quarter is "30:15", 30 is "30" without minutes

Reply:
Yes there isn't a date format but the client doesn't know how pass the value (ex: "30:15"). Can you put the example and a description in the swagger please?